commit 0f91abd5c68eb044d09733d18ef0f6b8ed128200
Author: Julien Cristau <jcristau@debian.org>
Date:   Thu Jul 19 20:37:26 2007 -0400

    Fix alpha build failures
    
    Don't include <asm/pci.h> in os-support/linux/lnx_axp.c, use "lnx.h" and
    <unistd.h> instead

diff --git a/hw/xfree86/os-support/linux/lnx.h b/hw/xfree86/os-support/linux/lnx.h
index 5aca9e9..085c942 100644
--- a/hw/xfree86/os-support/linux/lnx.h
+++ b/hw/xfree86/os-support/linux/lnx.h
@@ -11,7 +11,6 @@ extern int iopl __P ((int __level));
 
 /* new pciconfig_iobase syscall added in 2.2.15 and 2.3.99 */
 #  include <linux/unistd.h>
-#  include <asm/pci.h>
 extern long (*_iobase)(unsigned, int, int, int);
 
 /*
diff --git a/hw/xfree86/os-support/linux/lnx_axp.c b/hw/xfree86/os-support/linux/lnx_axp.c
index 0a76129..e255118 100644
--- a/hw/xfree86/os-support/linux/lnx_axp.c
+++ b/hw/xfree86/os-support/linux/lnx_axp.c
@@ -112,8 +112,8 @@ lnxGetAXP(void)
 /*
  * pciconfig_iobase wrappers and dynamic i/o selection
  */
-#include <linux/unistd.h>
-#include <asm/pci.h>
+#include "lnx.h"
+#include <unistd.h>
 #include <errno.h>
 
 /* glibc versions (single hose only) */
